### ISSUE TITLE
int.__hash__ raises exception for values outside of i64

### ISSUE THREAD

**Issue by ddjoh** [actonlang/acton#1348]
### Acton Version

0.15.0.20230602.13.43.27

### Steps to Reproduce

```
actor main(env):
    print(hash(2**131))
    await async env.exit(0)
```

### Expected Behavior

Printed hash value

### Actual Behavior

`internal error: overflow in converting int to bounded int`

**Comment by plajjan** [comment]
@sydow you fixed this in your #1367 PR, right?

**Comment by sydow** [comment]
Fixed by #1367.
